### ISSUE TITLE
Proposal: Bug Cleanup Triage

### ISSUE THREAD

**Issue by mickymkumar** [magma/magma#15437]
# Problem

There are 95 open bugs. The lack of a systematic approach to triaging and resolving these bugs is making worse  for the problem, leading to delays in development and deployment of new features.

# Solution

**Review and Categorization**

- **Review:** Create tracker board and review all bugs ticket issues
- **Labeling Needed**: If the bugs needs a label, assign the appropriate label.
    - **New Bugs:** All new bugs
    - **Critical:** Bugs that cause system crashes, data loss, or severe security vulnerabilities. These require immediate attention and resolution.
    - **High Priority:** Bugs that significantly impact functionality but do not cause complete system failure. These should be addressed promptly to maintain system performance and reliability.
    - **Medium Priority:**  Bugs that affect non-critical functions or have reasonable workarounds. These should be resolved in a timely manner but are not as urgent as high-priority bugs.
    - **Low Priority:** Minor issues or cosmetic bugs with minimal impact on functionality. These can be addressed as time permits.
    - **Needs Investigation:** Bugs that require further analysis to understand their impact, reproducibility, or the best approach for resolution. These need more information before they can be prioritized accurately.
    - **Completed:** bug is completed and closed.
- **Further Study Required**: If the bug requires further study by another community member, document the necessary work.

# Non-goals

- This proposal does not aim to address feature requests or new feature development.
- It does not include enhancements or performance optimizations unrelated to bug fixes.
- The proposal does not cover bugs in third-party dependencies unless they significantly impact Magma's core functionalities.
- Implementation of bug fixes

# Bid:

The proposed budget of $2000.

# Acceptance Criteria
The proposal will ensure a robust and efficient bug creation and organization system that enhances productivity and improves overall software quality. The acceptance criteria for this proposal is to Triage all bugs in correct board column.
Acceptance include:

- Creatation of project board
- study each bug
- add bugs to the correct board column

**Comment by lucasgonze** [comment]
Mickey to tighten up acceptance.
We'll call for a vote next week.

**Comment by lucasgonze** [comment]
Motion: @lucasgonze 
Second: @jblakley

**Comment by jordanvrtanoski** [comment]
+1

**Comment by Som-BARNS** [comment]
+1

**Comment by lucasgonze** [comment]
Goes to LF TAB next. @mrsjackson76 to forward.

**Comment by lucasgonze** [comment]
Still blocked on LF TAB. Som to document status.

**Comment by lucasgonze** [comment]
Discussion with Mickey Nov 27 2024: because this still hasn't bee pre-approved by TAB and LF and had a PO issued, this is being dropped.

**Comment by mickymkumar** [comment]
The Bug Tracker is below url:
[https://github.com/orgs/magma/projects/28/](https://github.com/orgs/magma/projects/28/ )

**Comment by mickymkumar** [comment]
Acceptance Criteria has been met:

- [x] Creation of project board **- [https://github.com/orgs/magma/projects/28/](https://github.com/orgs/magma/projects/28/)**
- [x] study each bug  **- All bugs have been looked at while adding to the board**
- [x] add bugs to the correct board column **- Currently all bugs are in open board column - further study by another community member as needed**

**Comment by jblakley** [comment]
+1 -- Jim
